### DISCUSSION TITLE
jitpack

### DISCUSSION THREAD

**Discussion by ligi** [LouisCAD/Splitties#267] [General]
I would love to consume this lib via jitpack - wonder if anyone found a way to do so. Unfortunately adding the repo is no option for me as I want to publish to f-droid and AFAIK this will not work when I add the repo which is most likely not whitelisted.

**Comment by LouisCAD** [comment]
mavenCentral will work, right?

**Reply by ligi** [reply]
Ah - it's also on MavenCentral - yea this works - did not see this in the README - thanks then!

**Reply by LouisCAD** [reply]
Nope, it's not on MavenCentral yet, only jcenter for now, but I plan on putting all versions there this year (I already had the domain com.louiscad + auto-renew and have already validated ownership for OSSRH/Sonatype).
